docs/preview/features/databricks/databricks-info-provider.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Interact with Databricks to gain insights"
+layout: default
+---
+
+# Interact with Databricks to gain insights
+
+## Installation
+
+To use these features, you have to install the following package:
+
+	```shell
+	PM > Install-Package Arcus.BackgroundJobs.Databricks
+	```
+
+## Usage
+The background job makes use of the available `DatabricksInfoProvider` to measure and report the finished job outcomes.
+This provider can be used outside the background job environment (ex. Azure Functions).
+
+```csharp
+ILogger logger = ...
+using (var client = DatabricksClient.CreateClient("https://databricks.base.url", "security.token"))
+using (var provider = new DatabricksInfoProvider(client, logger))
+{
+}
+```
+The provider provides two methods.
+**Getting finished jobs**
+Gets all the finished job runs within a given time window.
+```csharp
+DatabricksInfoProvider provider = ...
+var startOfWindow = DateTimeOffset.UtcNow.AddDays(-1);
+var endOfWindow = DateTimeOffset.UtcNow;
+IEnumerable<JobRun> finishedJobRuns = await provider.GetFinishedJobRunsAsync(startOfWindow, endOfWindow);
+// Custom 'JobRun' model.
+JobRun jobRun = finishedJobRuns.First();
+string runId = jobRun.Run.RunId;
+string jobId = jobRun.Run.JobId;
+string jobName = jobRun.JobName;
+RunResultState? resultState = jobRun.Run.State.ResultState;
+```
+
+**Measure finished job outcomes**
+Measures the finished job runs by reporting the results as logging metrics.
+
+This method is an combination of the previously defined method (**Getting finished jobs**) and calling an `ILogger` extension provided in this package (`ILogger.LogMetricFinishedJobOutcome`) which will write the finished job runs `JobRun` instances as metrics.
+
+> :bulb: With using our [Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights](https://www.nuget.org/packages/Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights/), you can report these Databricks reports as metrics in Application Insights.
```

Comment:
Maybe move this up to [here](https://github.com/arcus-azure/arcus.backgroundjobs/pull/45/files#diff-b56b59f4f5fab91029853d3b394e6ec8R15) since this is an installation thing?

Reply:
Yep.